examples/js/interactive/SelectionBox.js

```diff
@@ -102,8 +112,6 @@ THREE.SelectionBox = ( function () {
 
 		} else if ( this.camera.isOrthographicCamera ) {
 
-			if ( startPoint.equals( endPoint ) ) endPoint.addScalar( Number.EPSILON ); // avoid invalid frustum
```

Comment:
This code is not necessary anymore because of your changes in `select()`?

Reply:
The change replaces this one, but I see the code indeed makes more sense in `updateFrustum`. Changed.